app/views/projects/_form.html.haml

```diff
@@ -23,18 +22,21 @@
             %a{ href: '#markdown-preview', role: :tab, data: { toggle: :tab } } Preview
         .btnbar
           = render 'shared/editor_buttons'
+
       .panel-body
         .tab-content
           #markdown-source.tab-pane.active.fade.in{ role: 'tab-pane' }
             .form-group
-              = f.text_area :description, rows: 20, placeholder: 'Detailed description of your idea. You can use **markdown**.', class: 'form-control input-lg markdown-source-text', id: 'project_description'
+              = f.text_area :description, rows: 20, placeholder: Project.description_template,
```

Comment:
I don't like the UX of a placeholder. There is no way to copy & paste this in any way, come back to it etc. How about we format this a bit in mardown with some examples and pre-populate it in the text field?

Reply:
That was my first approach, I thought it's annoying having to delete nearly all of it when entering your project description. 
Ok, let's try it.